.github/ISSUE_TEMPLATE/host.md

```diff
@@ -20,6 +20,7 @@ Please fill in information, organizers, and sponsors as you assemble them, and t
 * [ ] All organizers agree to follow the event guidelines and CNCF Code of Conduct
 * [ ] Minimum of 3 sponsors committed
 * [ ] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
+* [ ] Agree to offer a diversity scholarship for the event
```

Comment:
For the issue, as a basic requirement this should be 

[ ] Agree to offer 3 complimentary tickets to increase diversity for the event.

Each event could offer additional sponsorship in terms of lodging, airfare, childcare and other expenses, but scholarship isn't very clear and not all events will have the financial support to offer monetary support.